omod/src/main/webapp/pages/login.gsp

```diff
@@ -155,8 +165,9 @@
                             <p class="left">
                                 <label for="password">
                                     ${ ui.message("referenceapplication.login.password") }:
+                                    <i class="far fa-eye" style="float: right" id="togglePassword"></i>
```

Comment:
I think in-line styling (i.e. `style="Float: right"`) can get confusing, especially since we already have a stylesheet for the login page: https://github.com/openmrs/openmrs-module-referenceapplication/blob/db4f80bef85555ff9e540d971ea818cec106207d/omod/src/main/compass/sass/login.scss. Let's just refactor the styling directive into the stylesheet.

I also like @sherrif10 's suggestion of including the eye icon within the textbox. Seems like this could be achieved fairly easily with a negative margin like here: https://www.javascripttutorial.net/javascript-dom/javascript-toggle-password-visibility/

Reply:
> like @sherrif10 's suggestion of including the eye icon within the textbox

Yeah, @isears! I definitely agree with you!  However, I tried implementing this but the results were ugly.  
```
<p class="left">
   <label for="password">
         ${ ui.message("referenceapplication.login.password") }:
   </label>
   <input class="form-control form-control-sm form-control-lg form-control-md" id="password" type="password" name="password" placeholder="${ ui.message("referenceapplication.login.password.placeholder") }"/>
   <i class="far fa-eye" style="float: right" id="togglePassword"></i>
 </p>
```
Am going to retry again